vite-project-kabinet/src/Content/PaginatedList.jsx

```diff
@@ -14,13 +16,53 @@ export const PaginatedList = ({items, itemsPerPage}) => {
         }
     };
 
+    // Fetch funkce
+    const getPostsWithCategoryName = async () => {
+        const reqPosts = await fetch(`/assets/constants/posts.json`)
+        const posts = await reqPosts.json()
+
+        const reqCategories = await fetch(`/assets/constants/categories.json`)
+        const categories = await reqCategories.json()
+
+        //chcem získať pole objektov, kde bude každý objekt obsahovať aj konkrétny názov kategórie
+        
+        const updatedPosts = posts.map((post) => {
+            const postCategsArray = post.categories
+            const newPostCategsArray = postCategsArray.map((catID) => {
+                const categObject = categories.find((category) => category.id === catID )
+                return categObject ? categObject.name : "kategória sa nenašla"
+            })
+
+            // Aby program správne fungoval, každý článok musí obsahovať vždy práve jednu kategóriu.
+            // Kabinet má aktuálne pár článkov takých, kde sú pridelené 2 kategórie
+
+            // todo: Dá sa nastaviť WP tak, aby uživateľ mohol zakliknúť vždy len jednu kategóriu ?
```

Comment:
Nedá. Muselo by se vytvořit nové pole, které by fungovalo stejně jen s tímhle omezením. Znamenalo by to, že by se musely upravit hodnoty pro všechny články.